exe-common/src/sync.rs

```diff
@@ -143,13 +143,17 @@ fn net_sync_to<A: Api>(
         // Iterate to the last block in the previous epoch.
         let mut cur_hash = our_tip.hash.clone();
         loop {
-            let block_raw = storage.read_block(&cur_hash.into()).unwrap();
+            let block_raw = storage.read_block(&cur_hash.clone().into()).unwrap();
```

Comment:
I had to add `.clone()` here, because `cur_hash` reassignment became optional, and compiler were complaining that it might be same already moved value. I figured clone is ok, since hash is a very simple value, but I might be doing something stupid from Rust POV here.